FileInstaller/FileUtils.cpp

```diff
@@ -0,0 +1,53 @@
+#include <windows.h>
+#include <iostream>
+#include <shlwapi.h>
+#include <memory>
+#include <filesystem>
+
+#include "FileInstaller.hpp"
+
+namespace fs = std::filesystem;
+
+namespace FileUtils {
+	void copy_file(std::wstring const &source_file_path, std::wstring const &dest_file_path) {
+		if (!CopyFileW(source_file_path.c_str(), dest_file_path.c_str(), false)) {
```

Comment:
why ```bFailIfExists``` == false? You would want to fail in that case

Reply:
We checks if the file exists before coping so it won't give us any additional information but i changed it to be true.